src/framework/audio/internal/sequencer.h

```diff
@@ -26,23 +26,22 @@
 #include "clock.h"
 #include "isequencer.h"
 #include "imidiplayer.h"
-#include "rpc/irpctarget.h"
 
 namespace mu::audio {
-class Sequencer : public ISequencer, public async::Asyncable, public rpc::IRPCTarget
+class Sequencer : public ISequencer, public async::Asyncable
 {
 public:
     Sequencer();
 
     Status status() const override;
     async::Channel<Status> statusChanged() const override;
 
-    bool play() override;
+    void play() override;
     void pause() override;
     void stop() override;
-    void seek(unsigned long miliseconds) override;
+    void seek(uint64_t miliseconds) override;
     void rewind() override;
-    void setLoop(unsigned int from, unsigned int to) override;
+    void setLoop(uint64_t from, uint64_t to) override;
```

Comment:
Better to add some unit to arguments names (like fromMilisecond, toMilisecond)

Reply:
renamed